packages/editor/src/client/routes/articleEditor.tsx

```diff
@@ -191,17 +195,51 @@ export function ArticleEditor({id}: ArticleEditorProps) {
     }, blocksCharLength)
   }
 
-  function countRichTextBlocksChars(blocks: BlockValue[]) {
-    return blocks.reduce((charLength: number, block: BlockValue) => {
-      if (!(block.type === BlockType.RichText)) return charLength
-      return countRichtextChars(charLength, block.value)
-    }, 0)
+  function countRichTextBlocksChars(block: RichTextBlockListValue) {
+    return countRichtextChars(0, block.value)
+  }
+
+  function countTitleChars(block: TitleBlockListValue): number {
+    return (block.value.title as string).length + (block.value.lead as string).length
   }
 
-  function countTitle(blocks: BlockValue[]) {
+  function countQuoteChars(block: QuoteBlockListValue) {
+    return (block.value.quote as string).length + (block.value.author as string).length
```

Comment:
I think the `as string` should not be necessary here. I think quote and author can be `string | undefined` and you need to check first if they are undefined.